src/Psl/Dict/unique_scalar.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Dict;
+
+use function is_array;
+use function array_unique;
+
+/**
+ * Returns a new dict in which each value appears exactly once.
+ *
+ * @psalm-param iterable<array-key, scalar> $iterable
+ *
+ * @psalm-return array<scalar>
```

Comment:
we don't use `@psalm-` prefix, and we always try to keep the return type narrowed as much as possible 😉 

```suggestion
 * @template Tk of array-key
 * @template Tv of scalar
 *
 * @param iterable<Tk, Tv> $iterable
 *
 * @return array<Tk, Tv>
```